src/utils.js

```diff
@@ -243,10 +243,12 @@ function extractAddress(plainText) {
  * @return {number} The top distance
  */
 export const getKeyboardAvoidingViewTopDistance = () => {
-  if (Platform.OS === 'android') {
-    return getStatusBarHeight();
-  }
-  return 0;
+  const statusBarHeight = getStatusBarHeight();
+  const calculatedHeight = (Platform.OS === 'ios')
```

Comment:
Why did this change for iOS? It was 0 before and now it's bigger than the Android

Reply:
I couldn't pinpoint the reason for this, but there are two major suspects for this need:
- The change on the React Navigation from 4 to 6
- The removal of the iOS modal wrapper, since it's now quite different from what it used to be on older versions.

The lack of this added height was causing the buttons to hide under the keyboard whenever it was opened.